src/store/modules/keycodes/index.js

```diff
@@ -76,22 +121,22 @@ const mutations = {
   },
   enableSteno(state) {
     state.steno = true;
-    state.keycodes = generateKeycodes(isISO(), state.steno);
+    state.keycodes = generateKeycodes(getOSKeyboardLayout(), state.steno);
   },
   disableSteno(state) {
     state.steno = false;
-    state.keycodes = generateKeycodes(isISO(), state.steno);
-  },
-  enableIso(state) {
-    state.keycodes = generateKeycodes(isISO(), state.steno);
-    if (state.active === 'ANSI') {
-      state.active = 'ISO/JIS';
-    }
+    state.keycodes = generateKeycodes(getOSKeyboardLayout(), state.steno);
   },
-  disableIso(state) {
-    state.keycodes = generateKeycodes(isISO(), state.steno);
-    if (state.active === 'ISO/JIS') {
-      state.active = 'ANSI';
+  updateKeycodeNames(state) {
```

Comment:
I think this needs to go after state.keycodes as it relies on the current value of state.keycodes for the lookup.

Reply:
never mind. I'm misreading the diff.